tools/report.collector.in

```diff
@@ -543,45 +543,69 @@ iscfvartrue() {
     getcfvar $1 $2 $3 | egrep -qsi "^(true|y|yes|on|1)"
 }
 
+iscfvarfalse() {
+    getcfvar $1 $2 $3 | egrep -qsi "^(false|n|no|off|0)"
+}
+
 get_logfiles() {
     cf_type=$1
     cf_file="$2"
-    facility_var="logfacility"
 
     case $cf_type in
-	corosync)
+        corosync)
             if [ -f "$cf_file" ]; then
                 debug "Reading $cf_type log settings from $cf_file"
-                if iscfvartrue $cf_type to_syslog "$cf_file"; then
-                    facility_var=syslog_facility
+                # The default value of to_syslog is yes.
+                if ! iscfvarfalse $cf_type to_syslog "$cf_file"; then
+                    facility_cs=$(getcfvar $cf_type syslog_facility "$cf_file")
+                    if [ -z "$facility_cs" ]; then
+                        facility_cs="daemon"
+                    fi
+                    # Always include system logs (if we can find them)
+                    msg="Mark:pcmk:`perl -e 'print time()'`"
+                    logger -p $facility_cs.info $msg >/dev/null 2>&1
+                    sleep 2 # Give syslog time to catch up in case it's busy
+                    findmsg 1 "$msg"
```

Comment:
Since we now do this in two places, maybe we should functionize it, e.g. find_syslog $facility_cs.info